rudder-web/src/main/webapp/index.html

```diff
@@ -2,60 +2,122 @@
     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 <head>
-<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
-<title>Rudder - Login</title>
-<link href="/style/rudder-login.css" rel="stylesheet" type="text/css" media="screen" />
-<link href="/images/themes/ui-lightness/jquery.ui.all.css" rel="stylesheet" type="text/css" />
-
-<link href="/images/rudder-favicon.ico" rel="icon" type="image/x-icon" />
-<script type="text/javascript" src="/javascript/jquery/jquery-1.8.3.min.js"></script>
-
-<title>Rudder - Login</title>
+    <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>Rudder - Login</title>
+    <link href="/bootstrap-3.2.0-dist/css/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="/style/font-awesome-4.6.2/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+    <link href="/style/ionicons-2.0.1/css/ionicons.min.css" rel="stylesheet" type="text/css">
+    <link href="/style/login.min.css" rel="stylesheet" type="text/css">
```

Comment:
This will not be minified during build ( a part of my infinite todo list ...) 

@RaphaelGauthier We should not use minified file for our code, because we will forget a some time to minify after fixes (or at least I will forget :) )